cinn/optim/ir_copy.h

```diff
@@ -18,10 +18,14 @@
 #include <vector>
 
 #include "cinn/ir/ir.h"
-#include "cinn/ir/ir_schedule.h"
 #include "cinn/ir/lowered_func.h"
 
 namespace cinn {
+
+namespace ir {
+class ModuleExpr;
```

Comment:
What's the purpose for changes in this file?

Reply:
to skip a compilation error caused by the conflict between the `ginac` and `protobuf` library, which is triggered by `ir_schedule.h` when it adds the `schedule_desc.h` header that has a  protobuf-generated header `schedule_desc.pb.h`

Reply:
Suggest to find other way in the future